gguf_orpheus.py

```diff
@@ -12,10 +12,15 @@
 import asyncio
 
 # LM Studio API settings
-API_URL = "http://127.0.0.1:1234/v1/completions"
+# API_URL = "http://127.0.0.1:1234/v1/completions"
+API_URL = os.environ.get("API_URL", "http://127.0.0.1:1234/v1/chat/completions") 
```

Comment:
Using this endpoint will apply a chat template to the input. This might be OK, but it's better to double-check with the reference implementation or the authors what is the correct format to use for this model as it can cause poor quality if not correctly applied. It might also be different depending if the pre-trained or fine-tuned model is used.